akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -281,20 +295,20 @@ object ShardRegion {
    * Send this message to the `ShardRegion` actor to request for [[ShardRegionStats]],
    * which contains statistics about the currently running sharded entities in the
    * entire region.
-   * Intended for testing purpose to see when cluster sharding is "ready" or to monitor
-   * the state of the shard regions.
    *
+   * Intended for monitoring the state of the shard regions.
    * For the statistics for the entire cluster, see [[GetClusterShardingStats$]].
    */
-  @SerialVersionUID(1L) case object GetShardRegionStats extends ShardRegionQuery with ClusterShardingSerializable
+  @SerialVersionUID(1L) final case object GetShardRegionStats extends ShardRegionQuery with ClusterShardingSerializable
```

Comment:
`objects` are always final

Reply:
Isn't there even a warning about that? Or only in IntelliJ?

Reply:
Oops that was just me being forgetful. No warning.